Integrations/DomainTools_Iris/DomainTools_Iris.yml

```diff
@@ -0,0 +1,489 @@
+      description: Technical Contact Country
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.TechnicalContact.Email
+      description: Technical Contact Email
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.TechnicalContact.Name
+      description: Technical Contact Name
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.TechnicalContact.Phone
+      description: Technical Contact Phone
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.BillingContact
+      description: Billing Contact Info
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.BillingContact.Country
+      description: Billing Contact Country
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.BillingContact.Email
+      description: Billing Contact Email
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.BillingContact.Name
+      description: Billing Contact Name
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.BillingContact.Phone
+      description: Billing Contact Phone
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.EmailDomains
+      description: Email Domains
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Identity.AdditionalWhoisEmails
+      description: Additional Whois Emails
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Registration
+      description: DomainTools Registration Info about the Domain
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Registration.DomainRegistrant
+      description: Domain Registrant
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Registration.RegistrarStatus
+      description: Reistrar Status
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Registration.DomainStatus
+      description: Domain Active Status
+      type: Boolean
+    - contextPath: DomainTools.PivotedDomains.Registration.CreateDate
+      description: Create Date
+      type: Date
+    - contextPath: DomainTools.PivotedDomains.Registration.ExpirationDate
+      description: Expiration Date
+      type: Date
+    - contextPath: DomainTools.PivotedDomains.Hosting
+      description: DomainTools Hosting Info about the Domain
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Hosting.IPAddresses
+      description: IP Addresses Info
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Hosting.IPCountryCode
+      description: IP Country Code
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Hosting.MailServers
+      description: Mail Servers Info
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Hosting.SPFRecord
+      description: SPF Record Info
+      type: Unknown
+    - contextPath: DomainTools.PivotedDomains.Hosting.NameServers
```

Comment:
this seem to a list of objects, need to specify these here

Reply:
I don't understand the requested change here. Is there a type that is list? I only see string, number, boolean, date, and unknown as available types.

Reply:
what im saying is that the output looks like this:
![image](https://user-images.githubusercontent.com/31018228/66911678-b00cfa80-f019-11e9-8b07-afcb82589c79.png)
if you wont specify `domain.value` for example as an output, this wont be usable

Reply:
Hmm, talking to my PM we would like to keep it the way it is and our playbooks and documentation would give examples of how to interact with the items not explicitly listed. Would this be alright?

Reply:
sorry, but not really
without defining the context path properly, the integration won't be usable

Reply:
Maybe I'm missing something, but how does it make it unusable? The data is still there and you can still access it.

Reply:
it is not usable in sense that the user has to manipulate the output in order to use it, and that only if he first runs the command in the war room and look at the context data to look at how the output looks
this is not good ux